### ISSUE TITLE
Migrating to v4.0.0 | Permission denied

### ISSUE THREAD

**Issue by jackc94** [vmstan/gravity-sync#318]
**Issue Description**
I ran the update script as described on my existing installation on my secondary license-hole and performed a clean install on my primary pi-hole. Things appeared to run smoothly however when I come to execute the script on either instance using gravity-sync I get the following: 

`pi@3kl-dns-02:~ $ gravity-sync compare
∞ Initializing Gravity Sync (4.0.0)
✓ Loading gravity-sync.conf
✓ Detecting local Pi-hole installation
∞ Detecting remote Pi-hole installationLoad key "/etc/gravity-sync/gravity-sync.rsa": Permission denied
pi@10.165.11.8's password: 
`

**Configuration Details**
2x Raspberry Pi 3B
Linux 3kl-dns-02 5.15.32-v8+ #1538 SMP PREEMPT Thu Mar 31 19:40:39 BST 2022 aarch64

**Comment by vmstan** [comment]
You'll need to make sure your local user is the owner of the relocated RSA file.

```
sudo chown user:user /etc/gravity-sync/gravity-sync.rsa
sudo chown user:user /etc/gravity-sync/gravity-sync.rsa.pub
```

Where 'user' is the user you want to own it.

**Comment by jackc94** [comment]
Having tried everything I just purged the install on the secondary node (the one I upgraded) and started a fresh install… no issues at all then. Another observation… checking for version you get this output:
`Initializing Gravity Sync (4.0.0)
✓ Evaluating arguments: VERSION
» Running version: 4.0.0
» Update available: 4.0.0
∞ Gravity Sync VERSION exited after 1 seconds `
Despite them being the same version I ran the update command and still the same message when checking version.

**Comment by vmstan** [comment]
I almost didn't even build a migrator. My original intent was to just have people do a fresh config. Glad it worked for you.

I submitted a small edit to the migrator that should make permissions set correctly in the future.
